cves/CVE-2018-1305.yml

```diff
@@ -212,4 +233,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    It seems their issue was both a design mistake and a bit of an oversight on 
+    their part. Initally back in 2010 they believed they wanted to be able to
+    deploy servlets at different times. This still is true. But what they didn't
+    think was that in the future the servlets would have secuirty constraints placed
+    on them and this order would be exploitable. They mitigated this risk by forcing
```

Comment:
Do you think this is something they could've foresaw?

Reply:
I don't really think so. They would have to have known there would be special security patches for servlets as well as knowing they had to be loaded in a particular order.